horizon_bsn/bsndashboard/static/networktemplate/actions/create/create.action.service.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+(function() {
+  'use strict';
+
+  angular
+    .module('bsn.bsndashboard.networktemplate.actions')
+    .factory('bsn.bsndashboard.networktemplate.actions.create.service', createService);
+
+  createService.$inject = [
+    'bsn.bsndashboard.networktemplate.resourceType',
+    'horizon.app.core.openstack-service-api.bsnneutron',
+    'horizon.framework.widgets.toast.service',
+    '$modal',
+    'horizon.framework.util.actions.action-result.service',
+  ];
+
+  /**
+   * @ngDoc factory
+   * @name horizon.app.core.images.actions.createService
+   * @Description A service to open the user wizard.
```

Comment:
service to create network template?
probably copy paste corrections for documentation